docs/contributing/contributing.md

```diff
@@ -0,0 +1,121 @@
+---
+order: 1
+description: <add description>
+parent:
+  order: 9
+  title: Contributing Guide
+---
+# Contributing
+
+- [Contributing](#contributing)
+  - [Using this repo](#using-this-repo)
+  - [Reviewing technical content PRs](#reviewing-technical-content-prs)
+  - [Writing and contributing](#writing-and-contributing)
+  - [Folder structure](#folder-structure)
+  - [Who works on the tutorials?](#who-works-on-the-tutorials)
+  - [Viewing Tutorial Builds](#viewing-tutorial-builds)
+    - [Preview PRs on a Deployed Preview](#preview-prs-on-a-deployed-preview)
+    - [Preview Draft PRs on a Local Web Browser](#preview-draft-prs-on-a-local-web-browser)
+  
+
+Thank you for considering making contributions. We appreciate your interest in helping us to create and maintain awesome tutorials.
+
+To set up your environment for success, follow our [technical set up](technical-setup.md) guidelines.
+
+## Using this repo
+
+- To provide feedback, file an issue and provide abundant details to help us understand how we can make it better.
+- To provide feedback and a fix, you can make a direct contribution. This repo is protected since we provide the code and the docs to help you learn. If you're not a member or maintainer, fork the repo and then submit a pull request (PR) from your forked repo to master.
```

Comment:
Should a collaborator create their PR against `master` or `develop`? @fadeev

Reply:
Always `develop`.